packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.22.0`
+
+- Added: Unit tests for the MVS and JES functionality [#1632](https://github.com/zowe/vscode-extension-for-zowe/pull/1632).
```

Comment:
we can remove this from the zowe-explorer/CHANGELOG.md since it's in the FTP one